src/Dashboard/Data/FunctionInstanceSnapshot.cs

```diff
@@ -17,7 +17,7 @@ public class FunctionInstanceSnapshot
 
         public Guid Id { get; set; }
 
-        public Guid HostInstanceId { get; set; }
```

Comment:
I sent a mail to @MikeStall  and @mathewc with the details, but my testing this morning found an issue. The Dashboard uses 'host message queue' name to determine if it has already aborted that host. It expects this name to be different with every invocation of the host so it can mark it as an 'abort receipt' in blobs.

I'm looking now to keep the old behavior for `hostInstanceId` so that everything continues working with the Dashboard and instead rename my property `HostStableId` and use that exclusively for the blob lease ids. That should keep the dashboard behavior the same.